src/systems/filecoin_vm/runtime/runtime.go

```diff
@@ -275,11 +275,11 @@ func (rt *VMContext) _transferFunds(from addr.Address, to addr.Address, amount a
 
 // TODO: This function should be private (not intended to be exposed to actors).
 // (merging runtime and interpreter packages should solve this)
-func (rt *VMContext) SendToplevelFromInterpreter(input InvocInput, ignoreErrors bool) (
+func (rt *VMContext) SendToplevelFromInterpreter(input InvocInput, catchErrors bool) (
```

Comment:
i think this is a confusing interface, because it will be used with `true` and `false` which will be unclear to read.

```go
rt.SendToplevelFromInterpreter(invIn, true) // what is true?
rt.SendToplevelFromInterpreter(invIn, false) // what is false?
```
for clarity, things like this are clearer:

```go
// way one
rt.SendToplevelFromInterpreter(inv, CatchErrors)
rt.SendToplevelFromInterpreter(inv, ThrowErrors) // or PropagateErrors

// way two
rt.SendToplevelFromInterpreter(inv) // does not catch errors
rt.SendToplevelFromInterpreterCatchErrors(inv)
```